compiler/src/iree/compiler/Codegen/Common/TileInferenceUtils.cpp

```diff
@@ -113,11 +114,9 @@ inferWorkgroupTileMultiplesFromPackUnPack(
           ? packedMultiples
           : unPackedMultiples;
   LLVM_DEBUG({
-    DBGS() << "\nInferred " << op->getName() << " multiples:\nsrc: [";
-    llvm::interleaveComma(srcMultiples, llvm::dbgs());
-    llvm::dbgs() << "]\nresult: [";
-    llvm::interleaveComma(destMultiples, llvm::dbgs());
-    llvm::dbgs() << "]\n\n";
+    DBGS() << "\nInferred " << op->getName()
+           << " multiples:\nsrc: " << llvm::interleaved_array(srcMultiples)
+           << "\nresult: " << llvm::interleaved_array(destMultiples) << "\n\n";
   });
```

Comment:
I don't know why we have blank lines before and after the log, but I think using multiple `LDBG` is still better. Given that you're touching the code, can you help make the change?

```cpp
LDBG("\nInffered " << op->getName() << " multiples");
LDBG("src: " << llvm::interleaved_array(srcMultiples));
LDBG("result: " << llvm::interleaved_array(destMultiples) << "\n");
```

I prefer not having blank lines before and after the log, like below, but I don't have a strong option here.

```cpp
LDBG("Inffered " << op->getName() << " multiples");
LDBG("src: " << llvm::interleaved_array(srcMultiples));
LDBG("result: " << llvm::interleaved_array(destMultiples));
```

Reply:
I'm going to keep this as-is here and clean up separately in a new PR